### PR TITLE
Restructuring

### PR DESCRIPTION
**Pull request by amygimma**

### CODE REVIEW

File: inst/scripts/003_analysis_figures.R [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+
+polymod <- merge(polymod, polymod_mean_contacts, by = "part_id", all.x = TRUE)
+
+
+polymod[, hh_size := fcase(
+  hh_size == 1, "1",
+  hh_size == 2, "2",
+  hh_size == 3, "3",
+  hh_size == 4, "4",
+  hh_size > 4, "6+"
```
Comment (jarvisc1) [inline inst/scripts/003_analysis_figures.R]: Needs to be changed to 4+ for the label

File: inst/scripts/aaa_scale_boot_matrices.R [hunk 2]
```diff
@@ -0,0 +1,93 @@
+source(here::here("R/create_cm.R"))
```
Comment (jarvisc1) [inline inst/scripts/aaa_scale_boot_matrices.R]: These need to be removed I think?

**Review by jarvisc1 — CHANGES_REQUESTED** [review]
Just a couple of lines to be removed. Also a typo that I think was probably mine. Change those and it can be merged. I think some of the functions that I'd moved to one have been separated back out but that won't affect the code running

File: inst/scripts/003_analysis_figures.R (hunk 1 above)
Comment (amygimma) [reply inst/scripts/003_analysis_figures.R]: thanks!

File: inst/scripts/aaa_scale_boot_matrices.R (hunk 2 above)
Comment (amygimma) [reply inst/scripts/aaa_scale_boot_matrices.R]: yep

**Review by jarvisc1 — APPROVED** [review]

**Pull request merged**